MagicaVoxel-file-format-vox-extension.txt

```diff
@@ -23,7 +23,7 @@ STRING	: value
 =================================
 (c) ROTATION type
 
-store a row-major rotation in the bits of a byte
+store a row-major (XZY) rotation in the bits of a byte
```

Comment:
Is that right? Shouldn't it be "(XYZ)" ?

Reply:
> Is that right? Shouldn't it be "(XYZ)" ?

It's been a while since I've worked with MagicaVoxel, but if I remember correctly, this should be correct. At least from what I remember. If what not, this is [the code that worked for me](https://github.com/mchorse/blockbuster/blob/e95fc7c08b662b5f1ca221f73a002ca4720b1826/src/main/java/mchorse/blockbuster/api/formats/vox/VoxReader.java#L178) to read the correct rotation for my Minecraft mod.

Reply:
The following transformation works as intended: 
```
let vector = Vec3::new(
    x * matrix.m11 + y * matrix.m12 + z * matrix.m13,
    x * matrix.m21 + y * matrix.m22 + z * matrix.m23,
    x * matrix.m31 + y * matrix.m32 + z * matrix.m33,
);
```

This suggests an XYZ rotation matrix